contracts/javascore/centralized-connection/src/main/java/xcall/adapter/centralized/XCallCentralizeConnection.java

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
Admin could be named relayer mabye?

Reply:
This means, relayer would be admin in this contract?